blueprints/codes/eurocode/nen_en_1992_1_1_c2_2011/chapter_6_ultimate_limit_state/formula_6_71.py

```diff
@@ -0,0 +1,105 @@
+"""Formula 6.71 from NEN-EN 1992-1-1+C2:2011: Chapter 6 - Ultimate limit state."""
+
+from blueprints.codes.eurocode.nen_en_1992_1_1_c2_2011 import NEN_EN_1992_1_1_C2_2011
+from blueprints.codes.formula import Formula
+from blueprints.codes.latex_formula import LatexFormula
+from blueprints.type_alias import DIMENSIONLESS, MPA
+from blueprints.validations import raise_if_less_or_equal_to_zero, raise_if_negative
+
+
+class FormADot2CriteriaBasedOnStressRangeLHS(Formula):
+    """Class representing Left Hand Side of formula 6.71 for the calculation of the fatigue criteria based on stress range."""
+
+    label = "6.71 (LHS)"
+    source_document = NEN_EN_1992_1_1_C2_2011
+
+    def __init__(
+        self,
+        gamma_f_fat: DIMENSIONLESS,
+        delta_sigma_s_equ_n_star: MPA,
+    ) -> None:
+        """[:math:`Δσ_{Ed}`] Loading side of equation [:math:`MPa`].
+
+        NEN-EN 1993-1-1+C2:2011 art.6.8.5 - Formula (6.71)
+
+        Parameters
+        ----------
+        gamma_f_fat : DIMENSIONLESS
+            [:math:`γ_{F,fat}`] Partial factor for fatigue actions [:math:`-`].
+        delta_sigma_s_equ_n_star : MPA
+            [:math:`Δσ_{s,equ}(N*)`] Damage equivalent stress range for types of reinforcement and considering number of cycles N* [:math:`MPa`].
+
+        Returns
+        -------
+        None
+        """
+        super().__init__()
+        self.gamma_f_fat = gamma_f_fat
+        self.delta_sigma_s_equ_n_star = delta_sigma_s_equ_n_star
+
+    @staticmethod
+    def _evaluate(
+        gamma_f_fat: DIMENSIONLESS,
+        delta_sigma_s_equ_n_star: MPA,
+    ) -> MPA:
+        """Evaluates the left hand side formula, for more information see the __init__ method."""
+        raise_if_negative(gamma_f_fat=gamma_f_fat, delta_sigma_s_equ_n_star=delta_sigma_s_equ_n_star)
+        return gamma_f_fat * delta_sigma_s_equ_n_star
+
+    def latex(self) -> LatexFormula:
+        """Returns LatexFormula object for left hand side formula 6.71."""
+        return LatexFormula(
+            return_symbol=r"\Delta \sigma_{Ed}",
+            result=f"{self:.3f}",
+            equation=r"\gamma_{F,fat} \cdot \Delta \sigma_{s,equ} (N^*)",
+            numeric_equation=rf"{self.gamma_f_fat:.3f} \cdot {self.delta_sigma_s_equ_n_star:.3f}",
+            comparison_operator_label=r"=",
+        )
+
+
+class FormADot2CriteriaBasedOnStressRangeRHS(Formula):
```

Comment:
same for this one.

Reply:
My apologies, that should've been Form6Dot71 which got then copy pasted to multiple places. Fixed in next commit.